content/en/02-feature-focus/08-custom-project-structure.md

```diff
@@ -0,0 +1,34 @@
+---
+since: 3.0
+---
+
+# Custom project structure
+
+Some advanced users like having WordPress in its own directory or move plugins, themes or uploads in another directory. VersionPress supports some scenarios.
```

Comment:
We should stress that the directories must still be under the "project root" (where the Git repo is maintained).